test/fuzz/test_runner.py

```diff
@@ -152,6 +167,35 @@ def main():
         )
 
 
+def generate_corpus_seeds(fuzz_pool, build_dir, seed_dir, targets, n):
```

Comment:
```suggestion
def generate_corpus_seeds(*, fuzz_pool, build_dir, seed_dir, targets, n):
```

python is not type safe, so please use named arguments to not accidentally mix up the order of the args and to simplify review

Reply:
Done